dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/HttpEndpointBasedTokenMapSupplier.java

```diff
@@ -40,109 +39,160 @@
 
 public class HttpEndpointBasedTokenMapSupplier extends AbstractTokenMapSupplier {
 
-	private static final Logger Logger = LoggerFactory.getLogger(HttpEndpointBasedTokenMapSupplier.class);
-	
-	private static final String DefaultServerUrl = "http://{hostname}:8080/REST/v1/admin/cluster_describe";
-	private final String serverUrl;
-	private static final Integer NumRetries = 2;
+    private static final Logger Logger = LoggerFactory.getLogger(HttpEndpointBasedTokenMapSupplier.class);
 
-	public HttpEndpointBasedTokenMapSupplier(int port) {
-		this(DefaultServerUrl, port);
+    private static final String DefaultServerUrl = "http://{hostname}:8080/REST/v1/admin/cluster_describe";
+    private final String serverUrl;
+    private static final Integer NumRetriesSameNode = 2;
```

Comment:
Yeah, but you should group "static" variables in one place - don't mix them like that